### ISSUE TITLE
rviz crash

### ISSUE THREAD

**Issue by gitunit** [raulmur/ORB_SLAM#21]
hola raúl,
did you ever run into the following error? http://answers.ros.org/question/192272/possible-limitation-of-points-in-markerline_strip-and-rviz/

by now i had this problem several times, mostly this happened after loop closure when my rosbag run the second time. i know its a rviz problem but maybe you solved this already?

**Comment by raulmur** [comment]
Sorry, but I never had that error.

**Comment by gitunit** [comment]
ok. thanks. further searching revealed its a known issue in rviz. https://github.com/ras-sight/hratc2015_framework/issues/4

also i realised that despite this error, ORB_SLAM is still running in the background and one can simply open a new console window and re-run rviz and open the provided config file. then everything is back again. :)

**Comment by gitunit** [comment]
i've tested a little bit more and realized that this crash always happens when KFs reach 1024. then i reopen rviz again (as mentioned earlier, ORB_SLAM is still running) then the next chrash happens exactly when reaching KFs of 2048.
i thought this information maybe useful.

**Comment by lilouch** [comment]
@gitunit  I have also the same issue ! Did you find a way to solve this ?

**Comment by gitunit** [comment]
no, i haven't. i tried a self compiled version of rviz which is build on ogre 1.10 but still the same issue.

**Comment by HChengmeng** [comment]
@gitunit  @lilouch  I have  the same issue ! Did you find a way to solve this ?

**Comment by lilouch** [comment]
No still didn't find a solution..

**Comment by gitunit** [comment]
btw, here is a tipp: toggle off keyframe markers and this error will not happen ;) best solution for me so far. you can toggle it on after KF has reached 1024 or before. however if KF markers are on and the 1024th KF will be added, rviz will crash.
